test/resources/selinux.py

```diff
@@ -0,0 +1,119 @@
+from robot.libraries.BuiltIn import BuiltIn
+from libostree import remote_sudo_rc, remote_sudo
+from typing import List
+
+ACCESS_CHECK_MAP = {
+    "/var/lib/microshift/version": ["cat"],
+    "/etc/microshift/config.yaml.default": ["cat"],
+}
+
+CONTEXT_CHECK_MAP = {
+    "system_u:object_r:container_var_lib_t:s0": [
+        "/var/lib/microshift",
+        "/var/lib/microshift-backups",
+    ],
+    "system_u:object_r:kubelet_exec_t:s0": [
+        "/usr/bin/microshift",
+        "/usr/bin/microshift-etcd",
+    ],
+    "system_u:object_r:kubernetes_file_t:s0": [
+        "/etc/microshift",
+    ],
+}
+
+EXPECTED_FCONTEXT_LIST = [
+    "/etc/kubernetes(/.*)?",
+    "/etc/microshift(/.*)?",
+    "/exports(/.*)?",
+    "/usr/bin/microshift",
+    "/usr/bin/microshift-etcd",
+    "/usr/local/bin/microshift",
+    "/usr/local/bin/microshift-etcd",
+    "/usr/local/s?bin/hyperkube.*",
+    "/usr/local/s?bin/kubelet.*",
+    "/usr/s?bin/hyperkube.*",
+    "/usr/s?bin/kubelet.*",
+    "/var/lib/buildkit(/.*)?",
+    "/var/lib/cni(/.*)?",
+    "/var/lib/containerd(/.*)?",
+    "/var/lib/containers(/.*)?",
+    "/var/lib/docker(/.*)?",
+    "/var/lib/docker-latest(/.*)?",
+    "/var/lib/kubelet(/.*)?",
+    "/var/lib/lxc(/.*)?",
+    "/var/lib/lxd(/.*)?",
+    "/var/lib/microshift(/.*)?",
+    "/var/lib/microshift-backups(/.*)?",
+    "/var/lib/microshift.saved(/.*)?",
+    "/var/lib/ocid(/.*)?",
+    "/var/lib/registry(/.*)?",
+]
+
+
+def get_expected_ocp_microshift_fcontext_list() -> List[str]:
+    return EXPECTED_FCONTEXT_LIST
+
+
+def get_fcontext_list() -> List[str]:
+    context_list = "kubernetes_file_t|container_var_lib_t|kubelet_exec_t|container_t"
+    semanage_filter_cmd = f"semanage fcontext -l | grep -E  \"({context_list})\" | awk '{{print $1 }}'"
+    return remote_sudo(semanage_filter_cmd).split('\n')
+
+
+def get_denial_audit_log() -> List[str]:
+    ausearch_filter_cmd = "ausearch --input-logs -m avc | grep microshift"
+    stdout, rc = remote_sudo_rc(ausearch_filter_cmd)
+    if rc == 0:
+        return stdout.split('\n')
```

Comment:
`splitlines()` should avoid an empty blank entry in the list due to a trailing newline.

Reply:
Oh thank you, updating it